.github/workflows/test_models_java_tests.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow performs tests in Java.
```

Comment:
Why did we change this file?

Reply:
> Why did we change this file?

I think it came after merging from main. Need to revert it.